widgets/accordion/js/accordion.js

```diff
@@ -99,6 +100,13 @@ jQuery( function ( $ ) {
 				} else {
 					updateHash();
 				}
+				if ( initialScrollPanel > 0 ) {
```

Comment:
Couldn't this just be `if ( initialScrollPanel ) {`, or is that `> 0` needed?

Reply:
The `> 0` is just a bit of extra protection against silly values like `-1` etc. :suspect: